chart/etcd-backup-restore/templates/etcd-statefulset.yaml

```diff
@@ -71,11 +71,16 @@ spec:
           protocol: TCP
         resources:
 {{ toYaml .Values.resources.etcd | indent 10 }}
+        env:
+        - name: ENABLE_TLS
+          value: "{{ if .Values.backupRestoreTLS }}true{{ else }}false{{ end }}"
+        - name: BACKUP_ENDPOINT
+          value: "http{{ if .Values.backupRestoreTLS }}s{{ end }}://localhost:{{ .Values.servicePorts.backupRestore }}"
+        - name: FAIL_BELOW_REVISION_PARAMETER
+          value: "{{ if .Values.backup.failBelowRevision }}&failbelowrevision={{ int $.Values.backup.failBelowRevision }}{{ end }}"
```

Comment:
For my information, what are these for?

Reply:
Again, these are the values we pass to the bootstrap script from the helm chart, now that the script no longer resides in the chart. These 3 parameters are basically the "configurable" parts of the script, and these need to be passed to the script either via env vars or as 3 new data entries in the configmap itself.